docs/sources/mimir/references/configuration-parameters/index.md

```diff
@@ -3394,6 +3394,12 @@ bucket_store:
   # CLI flag: -blocks-storage.bucket-store.index-header-lazy-loading-concurrency
   [index_header_lazy_loading_concurrency: <int> | default = 0]
 
+  # (experimental) If enabled, store-gateway will persist a sparse version of
+  # the index-header to disk on construction and load sparse index-headers from
+  # disk instead of the whole index-header.
+  # CLI flag: -blocks-storage.bucket-store.index-header-sparse-persistence-enabled
+  [index_header_sparse_persistence_enabled: <boolean> | default = true]
```

Comment:
Let's make this disabled by default. We don't want to enable this everywhere, only explicitly in environments we're testing.